google-cloud-bigquery/lib/google/cloud/bigquery/table.rb

```diff
@@ -141,6 +141,10 @@ def time_partitioning_type
         # Sets the partitioning for the table. See [Partitioned Tables
         # ](https://cloud.google.com/bigquery/docs/partitioned-tables).
         #
+        # You can only set partitioning when creating a table as in
+        # the example below. BigQuery does not allow you to change partiting
```

Comment:
typo: "partiting"

But since you have to change it anyway, what about adapting the error message?

"BigQuery does not allow you to convert a non-partitioned table to a partitioned table."